packages/stable/src/types.ts

```diff
@@ -1660,44 +1656,12 @@ export interface ProjectResponse {
   name: ProjectName;
   urlName: UrlName;
   oidcConfiguration?: OidcConfiguration;
-  userProfilesConfiguration: UserProfilesConfiguration;
```

Comment:
Did we remove the `userProfilesConfiguration` from project response as well?

Reply:
Good catch! I will undo that.